text/3817-promote-aarch64-pc-windows-msvc-to-tier-1.md

```diff
@@ -0,0 +1,180 @@
+- Feature Name: promote-aarch64-pc-windows-msvc-to-tier-1
+- Start Date: 2025-05-22
+- RFC PR: [rust-lang/rfcs#3817](https://github.com/rust-lang/rfcs/pull/3817)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Promote aarch64-pc-windows-msvc to Tier 1 with Host Tools.
+
+# Motivation
+[motivation]: #motivation
+
+About [30% of Rust users use Windows][survey-2024], while the majority of these developers and their
+customers are using x64 hardware, the usage of Arm64 Windows has been growing since it was first
+made available in Windows 10, and has been accelerating, especially with the availability of the
+SnapDragon X processors.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+No changes required: Rust tooling for Arm64 Windows has been available for a while now so this
+doesn't affect the end user experience.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Tier 1 targets must adhere to the [Tier 1 Target Policy][tier-1-policy]. Going through these
+requirements point-by-point:
+
+> Tier 1 targets must have substantial, widespread interest within the developer community, and must
+> serve the ongoing needs of multiple production users of Rust across multiple organizations or
+> projects. These requirements are subjective, and determined by consensus of the approving teams.
+
+As mentioned above, Windows users comprise a substatial propotion of Rust developers, and Arm64
+hardware is increasingly being used by them and their customers.
+
+> The target maintainer team must include at least 3 developers.
+
+There is an existing [Windows team for Rust][windows-team] and, within Microsoft, Arm64 Windows is
+considered to be a Tier 1 target for internal Rust usage.
+
+> The target must build and pass tests reliably in CI, for all components that Rust's CI considers
+> mandatory.
+> The target must not disable an excessive number of tests or pieces of tests in the testsuite in
+> order to do so. This is a subjective requirement.
+
+[The `dist-aarch64-msvc` CI job has been running reliably for over 4 years now][promote-tier-2],
+and I have [new CI jobs working where Rust is built and tested on Arm64 Windows runners][ci-draft-pr].
```

Comment:
Suggestion: maybe also state that "no significantly more tests are disabled for `aarch64-pc-windows-msvc` compared to other Tier 1 Windows targets", AFAICT this is true based on https://github.com/search?q=repo%3Arust-lang%2Frust+ignore-aarch64-pc-windows-msvc&type=code and the CI draft PR.

Reply:
I've noted all the tests that were disabled, with links to PRs and relevant issues.